webapp/channels/src/i18n/de.json

```diff
@@ -4168,7 +4168,7 @@
   "more_channels.join": "Beitreten",
   "more_channels.joining": "Beitreten...",
   "more_channels.next": "Weiter",
-  "more_channels.noMore": "Keine Ergebnisse für \"{text}\"",
+  "more_channels.noMore": "Keine weiteren Kanäle, denen beigetreten werden kann",
```

Comment:
@pvev we don’t handle this directly, please update it in weblate.

Reply:
ok!